connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoRelation.scala

```diff
@@ -121,4 +122,15 @@ private[kusto] case class KustoRelation(kustoCoordinates: KustoCoordinates,
       mode
     } else "hash"
   }
+
+  // Used for cached results
+  override def equals(other: Any): Boolean = other match  {
+    case that: KustoRelation => kustoCoordinates == kustoCoordinates && query == that.query
+    case _ => false
+  }
+
+  override def insert(data: DataFrame, overwrite: Boolean): Unit = {
+    KustoWriter.write(Some(0), data, kustoCoordinates, authentication, writeOptions =
+      WriteOptions.apply(timeout = new FiniteDuration(90, TimeUnit.MINUTES)))
```

Comment:
What does this mean? after 90 minutes we raise a timeout? why 90?

Reply:
i actually don't know .. i recall having timeouts from databricks but i dont know if it is related .. anyway i guess we should remove this